src/sparseml/export/helpers.py

```diff
@@ -12,35 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
 import os
 import shutil
 from collections import OrderedDict
 from enum import Enum
 from pathlib import Path
-from typing import Callable, List, Optional, Union
-
-import onnx
+from typing import Any, Callable, Dict, List, Optional, Union
 
 from sparseml.exporters import ExportTargets
-from sparsezoo.utils.onnx import save_onnx
 
 
 __all__ = [
     "apply_optimizations",
     "create_deployment_folder",
     "AVAILABLE_DEPLOYMENT_TARGETS",
     "ONNX_MODEL_NAME",
+    "create_export_kwargs",
 ]
 
 AVAILABLE_DEPLOYMENT_TARGETS = [target.value for target in ExportTargets]
 ONNX_MODEL_NAME = "model.onnx"
 ONNX_DATA_NAME = "model.data"
 
+_LOGGER = logging.getLogger(__name__)
+
+
+def create_export_kwargs(
+    loaded_model_kwargs: Dict[str, Any], export_target: str = "deepsparse"
```

Comment:
added `export_target` argument for the future, in case we need different logic for different export types (moving beyond `onnx`)